src/js/player.js

```diff
@@ -4074,6 +4080,7 @@ Player.prototype.options_ = {
     'textTrackDisplay',
     'loadingSpinner',
     'bigPlayButton',
+    'liveTracker',
```

Comment:
yeah your probably right.

Reply:
Actually I think we always want `liveTracker` as a component. It would be hard to add it as a child, and have its `durationchange` handler run before the rest of the `controlBar`